server-phoenix/lib/helios_web/router.ex

```diff
@@ -66,6 +66,8 @@ defmodule HeliosWeb.Router do
   end
 
   scope "/", HeliosWeb do
+    pipe_through :browser
```

Comment:
will adding a pipe to `:browser` cause any unintended effects to the regular `/*path`? Maybe place this under a new scope?

Reply:
Based off of the docs, I'm actually not quite sure why `pipe_through :browser` was not already in there for that scope, so I'm keeping that there, but the `GoogleAuthController` now has its own scope

Reply:
If we're adding this for a reason outside the commit message it would make sense to put it on its own commit.

Reply:
> I'm actually not quite sure why `pipe_through :browser` was not already in there for that scope

I believe `:browser` is a builtin pipe that was auto generated, and we never _needed_ to use it, but it looks like it adds some extra security, so yeah we should probably add it to the `/*pages` scope.